modules/ROOT/pages/troubleshoot-api-provisioning-issues.adoc

```diff
@@ -52,3 +52,45 @@ To resolve this issue:
 . Delete the API.
 . Resolve the username and password error.
 . Re-create the API.  
+
+[[api-spec-file-size-exceeded]]
+== API Specification File Exceeds Supported Size
+
+When provisioning an API with a specification file that is larger than 550 KBs, you might encounter issues with the provision operation or the Service Mesh services.
```

Comment:
Before Mysti left, we were working on a new template for troubleshooting, because this one is overloaded and repetitive. We believe that a troubleshooting topic need state only the issue or error visible to the audience and how to fix it.

In this case, you have already stated several times that the spec file cannot exceed 550 KB, so I'm not sure why you are supplying this topic at all.

But aside from that, consider....

== Troubleshooting <this issue or this error message>

As documented in the information about configuring Anypoint Service Mesh to automatically create an API in Anypoint Platform, your specification file must not exceed 550 KB. If it does, you receive error messages like the following:

Reply:
Could you share the new template for troubleshooting? Or it isn't ready to use yet?

Reply:
Hi @kbostrom , thanks for your recommendation.

I would recommend to check with Priya and would leave the change in the format of EVERY runbook to a different PR.